### PR TITLE
Added an FAQ about choosing between custom and base DC

### PR DESCRIPTION
**Pull request by kmoscoe**

### CODE REVIEW

File: custom_dc/index.md [hunk 1]
```diff
@@ -30,9 +30,10 @@ If you have the resources to develop and maintain a custom Data Commons instance
 
 For the following use cases, a custom Data Commons instance is not necessary:
 
-- You only want to make your own data available to the base public Data Commons site and don't need to test it. In this case, see the procedures in [Data imports](/import_dataset/index.html).
+- You only want to make your own data available to the base public Data Commons site. In this case, see the procedures in [Data imports](/import_dataset/index.html).
```
Comment (premr) [inline custom_dc/index.md:33]: should we simplify the wording to "You want to share your data publicly on the public Data Commons.

Comment (kmoscoe) [reply custom_dc/index.md:33]: Sure.

**Review by premr — APPROVED** [review]

**Review by premr — APPROVED** [review]
